scss/modules/_filters.scss

```diff
@@ -251,6 +251,17 @@ $filter-button-width: u(4.6rem);
   color: $green-dark;
 }
 
+.filter__message--error {
+  font-size: u(1.3rem);
+  color: $error;
+
+  button {
+    font-size: u(1.3rem);
```

Comment:
Let's add a `border-bottom: 1px dotted $error` to this button so it's clear it's a link.